### ISSUE TITLE
[Picker Input]: "Clear" button persists after performing clear action when `emptyValue` is an empty array

### ISSUE THREAD

**Issue by vasilii-kovalev** [epam/UUI#2472]
## Description

When passing an empty array to `emptyValue`, after clicking on the clear button (that removes all the selected values at once), it persists while it should disappear.

## Steps to Reproduce

1. Open "Getting selected entity" example on the component's documentation page: https://uui.epam.com/documents?id=pickerInput&mode=doc&category=components#getting_selected_entity (only this example currently has `emptyValue={[]}`)
2. Select several options
3. Click on the component's "Clear" button
4. Click on the picker to start loading options
5. Check console in the DevTools

## Actual result

The "Clear" button persists while it should disappear.
The following error is displayed in the console:

```code
FetchingHelper.ts:230
LazyTree: api does not returned requested items. Check that you handle 'ids' argument correctly.
Read more here: https://github.com/epam/UUI/issues/89
```

Note: in case of array data source, the error doesn't appear in the console, but the button still persists.

## Expected result

The "Clear" button disappears. No errors in the console.

## Environment

* UUI version: 5.9.0
* Browser name: Google Chrome v.127.0.6533.120
* Operating System: Windows 10 64-bit

## Additional info

Works correctly in 5.6.0.

In `onValueChange` callback, the parameter is an empty array as expected, when the clear action is performed, so it seems some additional internal handlers work incorrectly with it.

A temporary workaround is to not assign an empty array to `emptyValue` property, but do it in `onValueChange` callback as a fallback, like this:
```typescript
// Option 1.
onValueChange={(valueNext = []) => {
  setValue(valueNext)
}}

// Option 2.
onValueChange={(valueNext) => {
  setValue(valueNext ?? [])
}}
```
Would probably require some fighting with TypeScript.

**Comment by SvetlankaKuzyaka** [comment]
@AlekseyManetov Similar incorrect behavior occurs when `emptyValue={[]}` is used for **PickerList**:

https://github.com/user-attachments/assets/ca6b3ed8-2da7-4444-a0fe-a08bbbdae00a

Should I create a separate bug report for this issue, or will it be fixed as part of the current one?

**Comment by NatalliaAlieva** [comment]
> @AlekseyManetov Similar incorrect behavior occurs when `emptyValue={[]}` is used for **PickerList**:
> 
>  chrome_nkG0lXkCX7.mp4 
> Should I create a separate bug report for this issue, or will it be fixed as part of the current one?

Hello Svetlana,
Thank you for your comment. There is no need to create a separate ticket; this case will be fixed within the current one.

**Comment by NatalliaAlieva** [comment]
Released in 5.9.1 version.
